src/Clock.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+class Clock extends React.Component {
+  state = {
+    date: new Date(),
+    name: 0,
+    visibility: false,
```

Comment:
maybe it could be better to set default visibility:true?

Reply:
move visibility variable and everything connected with it to the App and render the Clock component only if it's true